core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/codesystem/CodeSystemRestService.java

```diff
@@ -243,7 +244,7 @@ public void update(
 		final String commitComment = String.format("Updated Code System %s", shortNameOrOId);
 		
 		if (codeSystem.getUpgradeOf() != null) {
-			throw new BadRequestException("'upgradeOf' property cannot be set through code system update API");
+			LoggerFactory.getLogger(getClass()).warn("'upgradeOf' property update support will be dropped in 8.0. It is only present in 7.x version for backward compatibility to let pre-8 datasets migrate their custom made upgrade CodeSystems work as upgrades.");
```

Comment:
The message at the end becomes a bit vague; I think I understand what the entire thing means 😅, but it could be shortened to something like this:

```suggestion
LoggerFactory.getLogger(getClass()).warn("'upgradeOf' property update support is only present in version 7.x and will be dropped with version 8.0.");
```

Reply:
Yeah, true, thanks for the suggestion.